api/src/main/java/edu/cornell/mannlib/vitro/webapp/edit/n3editing/VTwo/MultiValueEditSubmission.java

```diff
@@ -336,6 +333,23 @@ public void addLiteralToForm(EditConfigurationVTwo editConfig, FieldVTwo field,
             log.debug("could not find value for parameter " + var  );
         }
     }
+
+    private boolean isLangSetInFirstLiteral(EditConfigurationVTwo editConfig) {
+        Map<String, List<Literal>> literalsInScope = editConfig.getLiteralsInScope();
+        if (!literalsInScope.containsKey(LABEL)) {
+            return false;
+        }
+        List<Literal> labelLiterals = literalsInScope.get(LABEL);
+        if (labelLiterals.size() == 0) {
+            return false;
+        }
+        Literal literal = labelLiterals.get(0);
+        if (literal == null){
+            return false;
+        }
+        
+        return !StringUtils.isBlank(literal.getLanguage());
```

Comment:
`StringUtils.isBlank` will return true for and whitespace character. That will likely cause an issue the same as an empty string. Additionally, I would recommend the negation StringUtils method `isNotEmpty`